### PR TITLE
Update TOP solar radiation parameterization

### PR DESCRIPTION
**Pull request by daleihao**
Minor modification for TOP solar radiation parameterization: 
Use `STD_ELEV` std_elev when `STDEV_ELEV` is not available in fsurdata.

[BFB]

### CODE REVIEW

**Comment by rljacob** [comment]
Does this change answers?

**Comment by daleihao** [comment]
> Does this change answers?

No. It will affect the simulations only when TOP is turned on by use_top_solar_rad = .true..

**Comment by rljacob** [comment]
But isn't TOP on by default?  Its used in v3.

**Comment by daleihao** [comment]
It is just turned on in the water cycle simulations. It is off in the
master branch.

Robert Jacob ***@***.***> 于2024年6月12日周三 13:02写道：

**Comment by rljacob** [comment]
No it is on in the master branch.  Our policy is that the v3.0 tag should make v3.0 answers without relying on a runscript.   See https://github.com/E3SM-Project/E3SM/pull/6108.

File: components/elm/src/biogeophys/SurfaceAlbedoMod.F90 [hunk 1]
```diff
@@ -1811,7 +1811,7 @@ subroutine Albedo_TOP_Adjustment(bounds, num_pft, filter_pft, &
         lon_180 = lon(g)
          if (lon_180 > pi) lon_180 = lon_180-2._r8*pi    
     
-         if (cosz > 0._r8 .and. abs(lat(g)) < 1.047_r8 .and. stdev_elev(g) > 0._r8) then
```
[the remark this answers is not in the document]
Comment (daleihao) [reply components/elm/src/biogeophys/SurfaceAlbedoMod.F90]: Done.

**Comment by daleihao** [comment]
> No it is on in the master branch. Our policy is that the v3.0 tag should make v3.0 answers without relying on a runscript. See #6108.

After undoing one change, the code will not change the answers.

**Comment by rljacob** [comment]
discussion:  this makes sure TOP works for grids not used in WCYCLE v3.

**Comment by rljacob** [comment]
@bishtgautam please re-review.

**Comment by rljacob** [comment]
@bishtgautam please re-review.

**Review by bishtgautam — APPROVED** [review]

**Comment by rljacob** [comment]
@bishtgautam can this be merged?

**Comment by bishtgautam** [comment]
I will merge this today.

**Pull request merged**